Duplicati/Library/Modules/Builtin/run-script-example.sh

```diff
@@ -133,19 +133,19 @@ LOCALPATH=$DUPLICATI__LOCALPATH
 if [ "$EVENTNAME" == "BEFORE" ]
 then
 	# If the operation is a backup starting, 
-	# then we check if the --volsize option is unset
+	# then we check if the --dblock-size option is unset
 	# or 50mb, and change it to 25mb, otherwise we 
 	# leave it alone
 	
 	if [ "$OPERATIONNAME" == "Backup" ]
 	then
-		if [ "$DUPLICATI__volsize" == "" ] || ["$DUPLICATI__volsize" == "50mb"]
+		if [ "$DUPLICATI__dblock-_size" == "" ] || ["$DUPLICATI__dblock-_size" == "50mb"]
 		then
 			# Write the option to stdout to change it
-			echo "--volsize=25mb"
+			echo "--dblock-size=25mb"
 		else
 			# We write this to stderr, and it will show up as a warning in the logfile
-			echo "Not setting volumesize, it was already set to $DUPLICATI__volsize" >&2
+			echo "Not setting volumesize, it was already set to $DUPLICATI__dblock-_size" >&2
```

Comment:
Should this be `DUPLICATI__dblock_size` instead of `DUPLICATI__dblock-_size`?

See [here](https://forum.duplicati.com/t/duplicati-environment-variables/1645/2) for example.

Reply:
You are right, sorry. I corrected the this commit, with my last commit 50b0e2 .

Reply:
I do not understand [the build error](https://travis-ci.org/duplicati/duplicati/jobs/562449578#L8542)